pennylane/ops/qubit/attributes.py

```diff
@@ -152,6 +167,26 @@ def __contains__(self, obj):
 approximation.
 """
 
+has_unitary_generator_types = [
```

Comment:
This seems like a definite limitation of `Attributes`.

Reply:
What do you mean exactly?

Reply:
It'd be nice if we could just use the `has_unitary_generator` attribute, instead of having to maintain two lists. I'm just wondering what would be the effect if attributes stored a set of classes, instead of string names.